components/Accordian/index.jsx

```diff
@@ -0,0 +1,203 @@
+import Image from 'next/image';
+import React from 'react';
+
+const Accordian = props => {
+  const { jobTitle, location, exp } = props;
+  return (
+    <div className="mt-6 relative w-5/6 sm:w-2/3 overflow-hidden rounded-2xl shadow-md border">
+      <input
+        type="checkbox"
+        className="peer absolute top-0 inset-x-0 w-full h-20 opacity-0 z-10 cursor-pointer"
+      />
+      <div className="h-20 w-full pl-5 pt-2 flex items-start flex-col">
+        <h1 className="text-lg font-bold text-[#7A7A7A]">{jobTitle}</h1>
+        <div className="flex space-x-2 mt-2">
+          <div className="flex">
+            <div className="">
+              <Image
+                src="/images/Location Point.png"
+                width={20}
+                height={20}
+                alt="location"
+              />
+            </div>
+            <h1 className="text-sm font-normal text-[#7A7A7A] mx-2 ">
+              {location}
+            </h1>
+          </div>
+          <div className="flex">
+            <div className="">
+              <Image
+                src="/images/Travel bag.png"
+                width={20}
+                height={20}
+                alt="location"
+              />
+            </div>
+            <h1 className="text-sm font-normal text-[#7A7A7A] mx-2 ">{exp}</h1>
+          </div>
+        </div>
+      </div>
+
+      <div className="absolute top-7 right-5 text-[#7A7A7A] transition-transform duration-500 rotate-0 peer-checked:rotate-180">
+        {/* Arrow */}
+        <svg
```

Comment:
Shift the svg to separate file name ArrowIcon and place all the svg in a separate folder